features/autorisatie/bewoning/raadpleeg-bewoning-met-periode/autorisatie.feature

```diff
@@ -0,0 +1,264 @@
+    een nieuwe gemeentecode (92.12) staat die gelijk is aan de gemeentecode in de claim.
+
+    Abstract Scenario: Adres ligt in samengevoegde gemeente en <scenario>
+      Gegeven gemeente 'G1' heeft de volgende gegevens
+      | gemeentecode (92.10) | gemeentenaam (92.11) |
+      | 9999                 | Ons Dorp             |
+      En adres 'A3' heeft de volgende gegevens
+      | gemeentecode (92.10) | identificatiecode verblijfplaats (11.80) |
+      | 9999                 | 9999010000000003                         |
+      En de persoon met burgerservicenummer '000000012' is ingeschreven op adres 'A3' met de volgende gegevens
+      | gemeente van inschrijving (09.10) | datum aanvang adreshouding (10.30) |
+      | 9999                              | 20100818                           |
+      En de persoon is vervolgens ingeschreven op adres 'A2' met de volgende gegevens
+      | gemeente van inschrijving (09.10) | datum aanvang adreshouding (10.30) |
+      | 0518                              | 20200730                           |
+      En de persoon met burgerservicenummer '000000024' is ingeschreven op adres 'A3' met de volgende gegevens
+      | gemeente van inschrijving (09.10) | datum aanvang adreshouding (10.30) |
+      | 9999                              | 20200801                           |
+      En gemeente 'G1' is samengevoegd met de volgende gegevens
+      | nieuwe gemeentecode (92.12) | datum beëindiging (99.99) |
+      | 0800                        | 20230526                  |
+      Als bewoning wordt gezocht met de volgende parameters
+      | naam                             | waarde             |
+      | type                             | BewoningMetPeriode |
+      | datumVan                         | <datum van>        |
+      | datumTot                         | <datum tot>        |
+      | adresseerbaarObjectIdentificatie | 9999010000000003   |
+      Dan heeft de response 1 bewoning
+
+      Voorbeelden:
+      | datum van  | datum tot  | scenario                                                                                                                                 |
+      | 2023-07-01 | 2023-08-01 | periode ligt na datum samenvoeging (adresseerbaar object ligt in gemeente 0800)                                                          |
+      | 2022-01-01 | 2023-01-01 | periode ligt voor datum samenvoeging (adresseerbaar object ligt in gemeente 9999)                                                        |
+      | 2023-01-01 | 2023-08-01 | periode overlapt de datum samenvoeging (adresseerbaar object ligt een deel van de periode in gemeente 0800 en een deel in gemeente 9999) |
+      | 2019-01-01 | 2020-01-01 | bewoner in gevraagde periode is vertrokken voor de samenvoeging                                                                          |
+
+
+  Regel: De actuele gemeente van inschrijving van bewoners is niet relevant
+
```

Comment:
Is dit relevant voor de functionaliteit? Moeten we dit niet verplaatsen naar de tests samen met de 2 scenario's hieronder?

Reply:
verplaatsen naar test